lib/ansible/plugins/doc_fragments/acme.py

```diff
@@ -76,6 +78,9 @@ class ModuleDocFragment(object):
       - "For safety reasons the default is set to the Let's Encrypt staging
          server (for the ACME v1 protocol). This will create technically correct,
          but untrusted certificates."
+      - "The default value is U(https://acme-staging.api.letsencrypt.org/directory).
+         Note that in Ansible 2.14, this option I(will be required) and will no longer
```

Comment:
The `I(...)` syntax is for option/parameter names - see https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation. I vote against using it for emphasis. How about `C(...)` (code formatting)?

Reply:
I've replaced `I(...)` in this context with `*...*` (it was already used this way several times). Or is it better to remove all emphasis?

Reply:
`*. . .*` works for me. Thanks @felixfontein.